tests/e2e/specs/out-of-game/lobby.spec.js

```diff
@@ -413,3 +413,43 @@ describe('Lobby - P1 Perspective', () => {
     cy.get('[data-opponent-hand-card]').should('have.length', 6);
   });
 });
+
+describe('Redirecting to lobby from URL', () => {
+  
+  beforeEach(() => {
+    cy.wipeDatabase();
+    cy.visit('/');
+    cy.signupPlayer(myUser);
+    cy.createGamePlayer({ gameName: 'Test Game', isRanked: false }).then((gameSummary) => {
+      cy.wrap(gameSummary).as('gameSummary');
+      // Sign up new (other) user and subscribe them to game
+      cy.signupOpponent(opponentOne);
+      cy.subscribeOpponent(gameSummary.gameId);
+    });
+  });
+  
+  
+  it('Joins Lobby from url', function () {
```

Comment:
Let's check that the so let subscriptions work here by reading both players up and confirming the game starts

Reply:
We can do that in both tests and in one of them have the opponent ready up before the player joins via lobby url (so we confirm we initialize the existing ready state) and then in the other one both players can ready up after the player joins via url so we can see responses to the ready action